department-of-veterans-affairs/va.gov-cms
# VACMS-19813-Testing existing script for reporting axe violations when testing in …

## Description

Closes #19813

## Testing done


## Screenshots


## QA steps

Verify accessibility violations contained in the `cypress_accessibility_violations.json` file in tugboat are reported as a comment on a pull request.

As user _uid_ with _user_role_
1. Do this
2. Then
3. Then validate Acceptance Criteria from issue

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
